crates/forge/src/init.rs

```diff
@@ -183,19 +184,33 @@ pub fn run(project_name: &str) -> Result<()> {
     let cairo_version = ScarbCommand::version().run()?.cairo;
 
     if env::var("DEV_DISABLE_SNFORGE_STD_DEPENDENCY").is_err() {
-        ScarbCommand::new_with_stdio()
-            .current_dir(&project_path)
-            .manifest_path(scarb_manifest_path.clone())
-            .offline()
-            .arg("add")
-            .arg("--dev")
-            .arg("snforge_std")
-            .arg("--git")
-            .arg("https://github.com/foundry-rs/starknet-foundry.git")
-            .arg("--tag")
-            .arg(format!("v{version}"))
-            .run()
-            .context("Failed to add snforge_std")?;
+        if cairo_version >= MINIMAL_SCARB_FOR_REGISTRY {
+            ScarbCommand::new_with_stdio()
+                .current_dir(&project_path)
+                .manifest_path(scarb_manifest_path.clone())
+                .offline()
+                .arg("add")
+                .arg("--dev")
+                .arg("snforge_std")
+                .arg("--version")
+                .arg(version)
```

Comment:
Should probably be something like this
```suggestion
                .arg(format!("snforge_std@{version}"))
```

Reply:
alright noted